aiida/engine/processes/calcjobs/tasks.py

```diff
@@ -35,7 +36,22 @@
 logger = logging.getLogger(__name__)
 
 
+@decorator
+def pause_task_for_disabled_computer(wrapped, _, args, kwargs):
+    node = args[0]
+    authinfo = node.computer.get_authinfo(node.user)
+
+    if not authinfo.enabled:  # the target computer is disabled for this user
+        raise plumpy.PauseInterruption(
+            ('pausing CalcJob<{node.pk}> due to Computer<{node.computer.name}>'
```

Comment:
Two things:
1. The tuple around the string is not necessary because it is already in the `PauseInterruption` constructor, right?
2. No need to incude the calcjob pk in the message since this gets sets on the node itself so it is clear to which one it applies. I would just say `'Paused because Computer<{node.computer.name}> is disabled for User<{node.user.email}>'`

Reply:
true. ok, done.